src/content/en/updates/2017/09/picture-in-picture.md

```diff
@@ -0,0 +1,108 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Let's discuss about a Web API that would allow websites to create a floating video window over the desktop.
+
+{# wf_updated_on: 2017-09-07 #}
+{# wf_published_on: 2017-09-07 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: Let's discuss about a Web API that would allow websites to create a floating video window over the desktop #}
+{# wf_blink_components: Blink>Media #}
+
+# Picture In Picture (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Since April 2017, Chrome for [Android O supports Picture In Picture]. It allows
+users to play a `<video>` element in a pinned window that isn't blocked by other
+windows, so that they can watch while doing other things.
+
+Here's how it works: Open Chrome, go to a website that contains a video and
+play it fullscreen. From there, press the Home button to go to your Android
+Home Screen and the playing video will automatically transition in Picture In
+Picture. That's all! Pretty cool right?
+
+<figure>
+  <img src="/web/updates/images/2017/09/picture-in-picture-hero.jpg"
+       alt="Android Picture in Picture photo">
+  <figcaption>
+    <b>Figure 1.</b>
+    Android Picture in Picture photo
+  </figcaption>
+</figure>
+
+It is but... what about desktop? What if the website want to control that
+experience?
+
+The good news is that a [Picture In Picture Web API] specification is being
+drafted as we speak. This spec aims to allow websites to initiate and control
+this behavior by exposing the following sets of properties to the API:
+
+- Notify the website when video enters and leaves Picture in Picture mode.
```

Comment:
Notify the website when a [add a] video enters and leaves Picture in Picture mode.

Reply:
Done.